spec/command/apply_template_spec.rb

```diff
@@ -0,0 +1,173 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Command::ApplyTemplate do
+  context "when any template does not exist" do
+    let!(:app) { dummy_test_app }
+
+    it "raises error" do
+      result = run_cpl_command("apply-template", "gvc", "rails", "unexistent", "-a", app)
```

Comment:
English - "Unexistent" is not a standard English word. The correct term to use is "nonexistent."

https://www.quora.com/How-do-we-use-the-words-nonexistent-inexistent-and-unexistent-with-distinctions

Reply:
Done in e0e3fb62ec632ea71aa631f3fcb01f2a57ffcc41.